### PR TITLE
Add details on contributing to the CoreDataStack

### PR DESCRIPTION
**Pull request by rcedwards**
This PR:
* Adds contributing details
* Adds a pull request template file
* Adds an issue template file  

Addresses #101

### CODE REVIEW

**Comment by wolfbrain** [comment]
:eyes: @randomstep

**Comment by randomstep** [comment]
Minor editorial comments only. Can be merged when you're ready.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Contributing to the BNR Core Data Stack
+
+We gladly welcome contributions from the community in the form of issues, suggestions, feature requests, and pull request.
```
Comment (jeremy-w) [inline .github/CONTRIBUTING.md]: request => requests

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Issues
+
+When opening a new issue please do the following:
+
+### Search for Previous Issues
+
+Search [closed issues](https://github.com/bignerdranch/coredatastack/issues?q=is%3Aissue+is%3Aclosed) to make sure your problem has not already been addressed.
+
+Also look in [open issue](https://github.com/bignerdranch/coredatastack/issues) to avoid duplicates.
```
Comment (jeremy-w) [inline .github/CONTRIBUTING.md]: issue => issues

**Pull request merged**